### DISCUSSION TITLE
Sh1mmer

### DISCUSSION THREAD

**Discussion by cooltoo37** [3kh0/ext-remover#593] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
sh1mmer is not working does anybody have a raw or built shim for octopus I could use thanks

**Comment by 3kh0** [comment, marked as answer]
Download a shim at [files.ultimatesrv.com](https://dl.sh1mmer.me/raw%20shims/) and build it with the [sh1mmer web builder](https://sh1mmer.me/builder)

**Reply by cooltoo37** [reply]
THANKS i thought it got patched yo 3kh0 can i get caught  like do they notice that i am using it i am going to use shi1mmer to just remove policies not dev or uneneroll my device
